File: safari-600.7-branch/LayoutTests/http/tests/cache/resources/partitioned-cache-echo-state.php
<?php
require_once '../../resources/portabilityLayer.php';

header('Content-Type: application/javascript');
header('Cache-Control: max-age=3600');

$tmpFile = sys_get_temp_dir() . "/" . "partitioned-cache-state";
?>
var response = '<?php print file_get_contents($tmpFile); ?>';

File: LayoutTests/http/tests/cache/resources/partitioned-cache-echo-state.py
#!/usr/bin/env python3

import os
import sys
import tempfile

file = __file__.split(':/cygwin')[-1]
http_root = os.path.dirname(os.path.dirname(os.path.abspath(os.path.dirname(file))))

tmpFile = os.path.join(tempfile.gettempdir(), 'cache-partitioned-cache-state')
file = open(tmpFile, 'r')

sys.stdout.write(
    'Content-Type: application/javascript\r\n'
    'Cache-Control: max-age=3600\r\n\r\n'
    f'var response = \'{file.read()}\';\n'
)

file.close()